mongo_feature.go

```diff
@@ -70,10 +127,53 @@ func (m *MongoFeature) Close() error {
 }
 
 func (m *MongoFeature) RegisterSteps(ctx *godog.ScenarioContext) {
+	ctx.Step(`^remove all documents from the database`, m.RemoveAllDataFromDatabase)
+	ctx.Step(`^remove all documents in the "([^"]*)" collection`, m.RemoveAllDataFromCollections)
+	ctx.Step(`^remove all documents in the following collections: "([^"]*)"`, m.RemoveAllDataFromCollections)
 	ctx.Step(`^the following document exists in the "([^"]*)" collection:$`, m.TheFollowingDocumentExistsInTheCollection)
 	ctx.Step(`^the document with "([^"]*)" set to "([^"]*)" does not exist in the "([^"]*)" collection$`, m.theDocumentWithSetToDoesNotExistInTheCollection)
 }
 
+func (m *MongoFeature) RemoveAllDataFromDatabase() error {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	deletedDocs, err := m.ResetDatabase(ctx, m.Database.Name())
+	if err != nil {
+		return err
+	}
+
+	if deletedDocs.Count == 0 {
+		return fmt.Errorf("no documents were deleted in database: %s", m.Database.Name())
+	}
+
+	return nil
+}
+
+func (m *MongoFeature) RemoveAllDataFromCollections(collectionNames string) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	sliceCollNames := strings.Split(strings.ReplaceAll(collectionNames, " ", ""), ",")
+
+	deletedDocs, err := m.ResetCollections(ctx, m.Database.Name(), sliceCollNames)
+	if err != nil {
+		return err
+	}
+
+	if deletedDocs.Count == 0 {
```

Comment:
(minor): I think you would want to check if the ```deleteddocs == nil``` ? otherwise this might panic ``` deletedDocs.Count ``` ?